week-3/Homework/mandatory/2-bush-berries.js

```diff
@@ -9,10 +9,22 @@
 
   Use the tests to confirm which message to return
 */
+ 
+
+function bushChecker(bushBerryColours) {
+  if (bushBerryColours.includes('neon', 'transparent')){
```

Comment:
This works for our test cases, but isn't quite solving the problem as it was described in the comments :) Rather than looking to see whether the array contains particular bad values, can you try re-doing this looking to see whether _everything_ in the array is pink?